oqs-template/generate.yml

```diff
@@ -1516,3 +1516,138 @@ sigs:
                     'pretty_name': 'ECDSA p521',
                     'oid': '1.3.9999.8.5.2',
                     'code_point': '0xfef5'}]
+  -
+    family: 'CROSS' # TODO: add real OIDs
+    variants:
+      # RSDP 128
+      -
+        name: 'CROSSrsdp128balanced'
+        pretty_name: 'CROSS-rsdp-128-balanced'
+        oqs_meth: 'OQS_SIG_alg_cross_rsdp_128_balanced'
+        oid: '0.0.0.0.0.0.0.0.0.1'
```

Comment:
Not so good choice of OID. We simply keep counting up the "free" OID space "1.3.9999.x.y" I'd suggest you simply use x=9 and count y up through the Cross variants (just as Mayo did with x=8). That alleviates us of having to go through some official OID-approval procedure.

Reply:
Thanks @baentsch, switched to OIDs from 1.3.9999.9.1 to 1.3.9999.9.18

Reply:
We just managed to get official OIDs from IANA, should we switch to those? @baentsch

Reply:
Ok OIDs for CROSS are now from 1.3.6.1.4.1.62245.2.1.1 to 1.3.6.1.4.1.62245.2.1.18 and I added myself as code owner for genrate.yml

Reply:
Thanks, @rtjk . Marking this resolved then. Would you want to re-base and/or investigate the CI failures first? Or would you want me to look into the failure reasons in CI?